src/installer/pkg/sfx/installers/rpm_scripts/host/after_remove.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+if [[ -L /usr/local/bin/dotnet ]]; then
+  rm /usr/local/bin/dotnet && \
```

Comment:
syntax error due to extra ` && \`

```sh
after_remove.sh: line 5: syntax error near unexpected token `fi'
after_remove.sh: line 5: `fi'
```

Reply:
> syntax error due to extra ` && \`
> 
> ```shell
> after_remove.sh: line 5: syntax error near unexpected token `fi'
> after_remove.sh: line 5: `fi'
> ```

Hmm, not sure how this got into commit. I don't see it in my clone. Let me prepare another PR.

Reply:
Fix in PR: https://github.com/dotnet/runtime/pull/54151